### DISCUSSION TITLE
Getting SyntaxError error when using mysql2 in ubuntu

### DISCUSSION THREAD

**Discussion by sujal8976** [sidorares/node-mysql2#2314] [General]
Hello,
I am encountering an error specifically in Ubuntu when using mysql2, whereas it runs without issues in Windows. When I attempted to use mysql, it worked fine, but with mysql2, it shows a SyntaxError in Ubuntu, and why?

![Screenshot from 2023-12-08 13-51-44](https://github.com/sidorares/node-mysql2/assets/144937819/f9f8ab25-2874-4b73-aa8b-62097ba2f9a1)
![Screenshot from 2023-12-08 13-52-09](https://github.com/sidorares/node-mysql2/assets/144937819/733446e3-02b7-45b1-963a-1887b4a22f78)

**Comment by sidorares** [comment]
what node version do you have on your Ubuntu machine?

**Reply by sujal8976** [reply]
**v12.22.9**

I updated node and it is solved.. I was didn't aware of that..

Thank you so much

**Reply by sidorares** [reply]
see comments in https://github.com/sidorares/node-mysql2/issues/1953
